src/Data/List/Lazy/Types.purs

```diff
@@ -211,6 +211,12 @@ derive instance newtypeNonEmptyList :: Newtype (NonEmptyList a) _
 derive newtype instance eqNonEmptyList :: Eq a => Eq (NonEmptyList a)
 derive newtype instance ordNonEmptyList :: Ord a => Ord (NonEmptyList a)
 
+instance eq1NonEmptyList :: Eq1 NonEmptyList where
+  eq1 (NonEmptyList lhs) (NonEmptyList rhs) = eq1 lhs rhs
```

Comment:
Was hoping this shorthand would work, but it does not:
```purs
eq1 = over NonEmptyList eq1
```

Reply:
I've seen a number of cases where omitting arguments in an instance doesn't work. But I think it's arguably better to include the arguments explicitly in any case.

Reply:
I'd argue that the shorthand version is better, since it protects you from making a mistake like this:
```purs
eq1 (NonEmptyList rhs) (NonEmptyList lhs) = eq1 lhs rhs
```